Public/Src/App/Bxl/Bxl.dsc

```diff
@@ -75,7 +75,6 @@ namespace Main {
             importFrom("BuildXL.FrontEnd").CMake.dll,
             importFrom("BuildXL.FrontEnd").Sdk.dll,
             importFrom("Newtonsoft.Json").pkg,
-            importFrom("System.Data.SQLite.Core").pkg,

```

Comment:
note that this package has the `.Core` suffix which is the one you didn't remove from `config.dsc` (I'm not sure if this comment carries any significance though :))

Reply:
our main exe doesn't need the dll to compile, so I removed it.

---
In reply to: [310281736](https://github.com/microsoft/BuildXL/pull/696#discussion_r310281736) [](ancestors = 310281736)